### ISSUE TITLE
Compilation Errors

### ISSUE THREAD

**Issue by angelocala94** [phonegap/phonegap-mobile-accessibility#16]
I have some mistakes in compilation.
I've started a new Ionic Project with `ionic start myApp blank`
After that, i've added iOS platform with `ionic platform add ios`
Now, i have installed phonegap-mobile-accessibility plugin with 
`cordova plugin add https://github.com/phonegap/phonegap-mobile-accessibility.git`
But this is the results: 
```
** BUILD FAILED **


The following build commands failed:
	Ld build/emulator/myApp_blank.app/myApp_blank normal i386
(1 failure)
ERROR building one of the platforms: Error: /Users/username/Desktop/myApp_blank/platforms/ios/cordova/build: Command failed with exit code 65
You may not have the required environment or OS to build this project
```
Where am i wrong?

cordova -v 4.2.0
ionic -v 1.3.22

**Comment by shazron** [comment]
See https://github.com/phonegap/phonegap-mobile-accessibility/issues/11
(since you are using an older version of cordova)

**Comment by majornista** [comment]
I believe this should be fixed with pull request #14.
